content/en/blog/_posts/2023-08-15-kubernetes-1.28-blog.md

```diff
@@ -26,13 +26,25 @@ Much like a garden, our release has ever-changing growth, challenges and opportu
 # What's New (Major Themes)
 
 ## Changes to supported skew between control plane and node versions
-This enables testing and expanding the supported skew between core node and control plane components by one version from n-2 to n-3, so that node components (kubelet and kube-proxy) for the oldest supported minor version work with control plane components (kube-apiserver, kube-scheduler, kube-controller-manager, cloud-controller-manager) for the newest supported minor version.
 
-This is valuable for end users as control plane upgrade will be a little faster than node upgrade, which are almost always going to be the longer with running workloads.
+Kubernetes v1.28 expands the supported skew between core node and control plane
+components by one minor version, from _n-2_ to _n-3_, so that node components
+(kubelet and kube-proxy) for the oldest supported minor version work with control plane components (kube-apiserver, kube-scheduler, kube-controller-manager, cloud-controller-manager) for the newest supported minor version.
 
-The Kubernetes yearly support period already makes annual upgrades possible. Users can upgrade to the latest patch versions to pick up security fixes and do 3 sequential minor version upgrades once a year to "catch up" to the latest supported minor version.
+This is valuable for end users as control plane upgrade will be a little faster than
+node upgrade, which are almost always going to be the longer with running workloads.
```

Comment:
```suggestion
node upgrades, which are almost always going to be longer with running workloads.
```

Reply:
Additionally, I feel like this sentence is confusing because to me, a non-native English speaker, it reads as below,

"This is valuable for end users as control plane upgrades, which are almost always going to be longer with running workloads, will be a little faster than node upgrades."

Could we make it more idiomatic, if possible?